translations/sk.json

```diff
@@ -299,7 +299,9 @@
     "verificationResult": "Výsledok overenia",
     "verified": {
       "title": "Parkovacie karty pridané",
-      "text": "Vaše parkovacie karty pre e-mailovú adresu **{{email}}** boli pridané. Zoznam všetkých kariet nájdete v časti Parkovacie karty.",
+      "text_empty": "Vaše parkovacie karty pre e-mailovú adresu **{{email}}** boli pridané. Zoznam všetkých kariet nájdete v časti Parkovacie karty.",
+      "text_0": "Vaše parkovacie karty pre e-mailovú adresu **{{email}}** boli pridané. Zoznam všetkých kariet nájdete v časti Parkovacie karty.\n\n Pridané vozidlo s EČV **{{licencePlates}}** nájdete a môžete spravovať v časti Vozidlá.",
+      "text_1": "Vaše parkovacie karty pre e-mailovú adresu **{{email}}** boli pridané. Zoznam všetkých kariet nájdete v časti Parkovacie karty.\n\n Pridané vozidlá s EČV **{{licencePlates}}** nájdete a môžete spravovať v časti Vozidlá.",
```

Comment:
Make sure that not listing `_2` key for "other" plurals has a predictable and correct behaviour (if it fallbacks to `_1`). If we are not sure, please list all possible plural keys (`one`, `few`, `other` / `1` ,`2`,`3`).